after_deploy.sh

```diff
@@ -4,7 +4,3 @@ REPOSITORY=https://github.com/thepracticaldev/dev.to_private/
 REVISION="$(git rev-parse HEAD)"
 COMMIT_AUTHOR=$(git --no-pager log -1 --pretty=format:'%an')
 PAYLOAD='{ "environment":"production", "repository":"'${REPOSITORY}'", "revision":"'${REVISION}'", "username":"'${COMMIT_AUTHOR}'"}'
-curl -X POST \
-  -H "Content-Type: application/json" \
-  -d "${PAYLOAD}" \
-  "https://airbrake.io/api/v4/projects/${AIRBRAKE_PROJECT_ID}/deploys?key=${AIRBRAKE_PROJECT_KEY}"
```

Comment:
You are correct we have this already setup via Heroku deploy hooks for our use case. Since deploying is usually very platform specific I think removing the entire script is likely the best idea. I didnt realize that the lines above were just for populating that curl command. Good call!

Reply:
A bit late to this but yep we definitely don't need after_deploy.sh anymore